package.json

```diff
@@ -29,23 +29,23 @@
 		"@types/prompts": "^2.0.9",
 		"@types/sinon": "^10.0.2",
 		"@types/source-map-support": "^0",
-		"@typescript-eslint/eslint-plugin": "^4.18.0",
```

Comment:
@types/node above is still set to 14.x

Reply:
Do we still need node-fetch as an external dependency with Node 18?

Reply:
a5bbd1e